packages/core/src/ui/widgets/colorBar/types/ColormapCanvasProps.ts

```diff
@@ -0,0 +1,14 @@
+import type { ColorBarImageRange } from './ColorBarImageRange';
+import type { ColorBarSize } from './ColorBarSize';
+import type { ColorBarVOIRange } from './ColorBarVOIRange';
+import type { Colormap } from './Colormap';
+
+export interface ColorBarCanvasProps {
+  colormap: Colormap;
+  size?: ColorBarSize;
+  imageRange?: ColorBarImageRange;
+  voiRange?: ColorBarVOIRange;
+
+  container?: HTMLElement;
+  showFullPixelValueRange?: boolean;
+}
```

Comment:
This is really the same as the colorBarCanvasProps

Reply:
actually the name seem to be wrong

Reply:
File deleted! I don't know why it was cloned after a refactoring.